e2e/tests/api-driven/src/flowStatusHistory/flowStatusHistory.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Flow status history
+
+  @regression @flow-status-history
+  Scenario: Creating a flow
+    When a new flow is added
+    Then the status of the flow is online by default
+    And a flow_status_history record is created
+
+  @regression @flow-status-history
+  Scenario: Updating a flow 
+    When a new flow is added
```

Comment:
nit: should this be `When an existing flow is updated` ?

Reply:
Fixed - this was just a shortcut to re-use a step but I've replaced both with `Given()`